git-config/tests/file/access/read_only.rs

```diff
@@ -64,7 +64,7 @@ fn get_value_for_all_provided_values() -> crate::Result {
             &[cow_str("")],
         );
 
-        assert_eq!(config.string("doesnt", None, "exist"), None);
+        assert_eq!(config.string("doesn't", None, "exist"), None);
```

Comment:
This is interesting as I didn't dare putting `'` into section names for query. But it's OK to do that and has the same effect, good to see.